src/foundation/variables.css

```diff
@@ -1 +1,180 @@
+
+  /* stylelint-disable value-keyword-case */
+  --font-family-sans-serif: -apple-system, blinkmacsystemfont, 'Segoe UI',
+    roboto, 'Helvetica Neue', arial, 'Noto Sans', sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
+  --font-family-monospace: sfmono-regular, menlo, monaco, consolas,
+    'Liberation Mono', 'Courier New', monospace;
+  --font-family-base: var(--font-family-sans-serif);
+  /* stylelint-enable value-keyword-case */
+
+  /* Property --font-size-root effects the value of `rem`, which is used
+   * for as well font sizes, paddings and margins
+   * Property --font-size-base effects the font size of the body text.
+   * It assumes the browser default, typically `16px`
+   */
+  --font-size-root: null;
+  --font-size-base: 1rem;
+  --font-size-lg: calc(var(--font-size-base) * 1.25);
+  --font-size-sm: calc(var(--font-size-base) * 0.875);
+  --font-weight-lighter: lighter;
+  --font-weight-light: 300;
+  --font-weight-normal: 400;
+  --font-weight-bold: 700;
+  --font-weight-bolder: bolder;
+  --font-weight-base: var(--font-weight-normal);
+  --line-height-base: 1.5;
+  --line-height-lg: 2;
+  --line-height-sm: 1.25;
+
+  /* Headings */
+  --h1-font-size: calc(var(--font-size-base) * 2.5);
+  --h2-font-size: calc(var(--font-size-base) * 2);
+  --h3-font-size: calc(var(--font-size-base) * 1.75);
+  --h4-font-size: calc(var(--font-size-base) * 1.5);
+  --h5-font-size: calc(var(--font-size-base) * 1.25);
+  --h6-font-size: var(--font-size-base);
+  --headings-margin-bottom: var(--spacer) / 2;
+  --headings-font-family: null;
+  --headings-font-style: null;
+  --headings-font-weight: 500;
+  --headings-line-height: 1.2;
+  --headings-color: null;
+
+  /* Display headings */
+  --display1-size: 6rem;
+  --display2-size: 5.5rem;
+  --display3-size: 4.5rem;
+  --display4-size: 3.5rem;
+  --display1-weight: 300;
+  --display2-weight: 300;
+  --display3-weight: 300;
+  --display4-weight: 300;
+  --display-line-height: var(--headings-line-height);
+
+  /* Text variations */
+  --lead-font-size: calc(var(--font-size-base) * 1.25);
+  --lead-font-weight: 300;
+  --small-font-size: 0.875em;
+  --sub-sup-font-size: 0.75em;
+  --text-muted: var(--gray-600);
+  --initialism-font-size: var(--small-font-size);
+
+  /* Blockquotes */
+  --blockquote-small-color: var(--gray-600);
+  --blockquote-small-font-size: var(--small-font-size);
+  --blockquote-font-size: calc(var(--font-size-base) * 1.25);
+
+  /* Horizontal rule */
+  --hr-color: inherit;
+  --hr-height: var(--border-width);
+  --hr-opacity: 0.25;
+  --hr-margin-y: var(--spacer);
+  --legend-margin-bottom: 0.5rem;
+  --legend-font-size: 1.5rem;
+  --legend-font-weight: null;
+  --mark-padding: 0.2em;
+  --mark-bg: #fcf8e3;
+  --dt-font-weight: var(--font-weight-bold);
+  --nested-kbd-font-weight: var(--font-weight-bold);
+  --list-inline-padding: 0.5rem;
+
+  /* Code */
+  --code-font-size: var(--small-font-size);
+  --code-color: var(--pink);
```

Comment:
It doesn't exist too.

Reply:
Thank you for pointing this out. I will add colour schema and missing variables.

Reply:
By the way, [var()](https://developer.mozilla.org/en-US/docs/Web/CSS/var) accepts default value as default value if custom property does not exist.

Should I provide default values everywhere, should not I?

Reply:
I would rather define those variables or not use var at all.